### ISSUE TITLE
single song download

### ISSUE THREAD

**Issue by Thrusterwin** [surenjanath/Spotify-Song-Downloader-PyQt5#5]
Is it possible to add one more feature if it is then please add a feature so that we can download a single song also

I mean, there are several playlists that have songs which we don't like 

Or maybe another feature like after pasting the playlist link we'll get a list preview and then we can deselect those songs which we don't like 

That'll be very helpful

**Comment by arthurnevelsteen** [comment]
Indeed, I wanted to download the last few songs of a playlist, but I had to download the full playlist which took a long time because there are 500 songs in it.

**Comment by surenjanath** [comment]
Okay will try and add it as a feature in next update :) 
Thanks. Will try and work on it by Weekend
